pypeit/scripts/collate_1d.py

```diff
@@ -0,0 +1,823 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script collates multiple 1d spectra in multiple files by object, 
+runs flux calibration/coadding on them, and produces files suitable
+for KOA archiving.
+"""
+
```

Comment:
So far we have had all our scripts import methods and classes from core or other directories, rather than in this module where the module in scripts also has the methods and classes it needs. I actually prefer your approach, but it depends on whether these methods are used elsewhere.

Reply:
I was planning to split up pypeit_collate_1d.py in the next pull release. Do you want me to do that in this one?

Reply:
Fine with me.